src/Gateway/Startup.cs

```diff
@@ -75,6 +75,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, IMessage
 
             app.UseMiddleware<IgnoreRoute>(ignoredRoutes);
             app.UseMiddleware<AddHeaders>();
+            app.UseErrorHandler();
```

Comment:
This should probably go at the start of the pipeline in case an exception occurs in the middleware.

Reply:
Sorted in latest commits